src/pages/workspace/invoices/WorkspaceInvoiceVBASection.tsx

```diff
@@ -164,7 +165,7 @@ function WorkspaceInvoiceVBASection({policyID}: WorkspaceInvoiceVBASectionProps)
     const addPaymentMethodTypePressed = (paymentType: string) => {
         hideAddPaymentMenu();
         if (paymentType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT || paymentType === CONST.PAYMENT_METHODS.BUSINESS_BANK_ACCOUNT) {
-            BankAccounts.openPersonalBankAccountSetupView();
+            BankAccounts.openPersonalBankAccountSetupView(undefined, isUserValidated);
```

Comment:
Just curious, why `undefined` and not an empty string or null?

Reply:
yeah it can be null or empty string, we need a falsely value there, so I choose undefined 😄